dataland-frontend/src/components/pages/CompanyInformation.vue

```diff
@@ -4,8 +4,28 @@
       <p class="font-medium text-xl">Loading company information...</p>
       <i class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
     </div>
-    <div v-else-if="companyInformation && !waitingForData" class="text-left company-details">
```

Comment:
text-left no longer necessary?

Reply:
yes, since I added the "left elements" for the company information. Nevertheless, the right element like the context menu should not flush left.

Reply:
Should the framework cards in the last incomplete row be left-aligned?
![image](https://github.com/d-fine/Dataland/assets/104447806/43230d79-59dd-4325-8096-acd5c6faafd7)
Currently they are shifted to be in the center. What do you think?

Reply:
right now, they are left aligned.

Reply:
there is another story for a restyling of the company cockpit page anyway: 
https://jira.d-fine.dev/browse/DALA-3419

Reply:
Added to https://jira.d-fine.dev/browse/DALA-2981?workflowName=Software+Simplified+Workflow+for+Project+DALA&stepId=1